src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="form_field">
+      <label class="search_bar search1" >
```

Comment:
Check the class names one more time, please
```suggestion
      <label class="search-bar search-bar--big">
```

Reply:
As for disappearing icon in big input, I cant understand why it happens, it is the same like in the small input, changed only width an height.

And I checked focus style for big input one more time, I think there might be changing font-weight, but in Figma I didn't saw it.

class names - done.